internal/audit/auditlogtextformatter.go

```diff
@@ -153,14 +153,17 @@ func groupAndPrint(vulnerable []types.Coordinate, nonVulnerable []types.Coordina
 func (f AuditLogTextFormatter) Format(entry *Entry) ([]byte, error) {
 	auditedEntries := entry.Data["audited"]
 	invalidEntries := entry.Data["invalid"]
+	excludedEntries := entry.Data["excluded"]
 	packageCount := entry.Data["num_audited"]
 	numVulnerable := entry.Data["num_vulnerable"]
+	numExcluded := entry.Data["num_exclusions"]
 	buildVersion := entry.Data["version"]
-	if auditedEntries != nil && invalidEntries != nil && packageCount != nil && numVulnerable != nil && buildVersion != nil {
+	if auditedEntries != nil && invalidEntries != nil && excludedEntries != nil && packageCount != nil && numVulnerable != nil && numExcluded != nil && buildVersion != nil {
```

Comment:
This is just an observation but this is UGLLLLY as it grows.

You might create a func like:

```
func isNil(params ...interface{}) bool {
  foundNil := false
  for _, v := range params {
     if v == nil || reflect.ValueOf(v).IsNil() {
       foundNil = true
     }
  }
  return foundNil
}
```

Then hypothetically you can just do `isNil(auditedEntries, invalidEntries, excludedEntries, packageCount, numVulnerable, numExcluded, buildVersion)`

This is pseudo code so no idea if it compiles, just an idea so we don't end up with a large finger bug some day :)